vars/kibanaPipeline.groovy

```diff
@@ -97,7 +97,7 @@ def uploadCoverageArtifacts(prefix, pattern) {
 def withGcsArtifactUpload(workerName, closure) {
   def uploadPrefix = "kibana-ci-artifacts/jobs/${env.JOB_NAME}/${BUILD_NUMBER}/${workerName}"
   def ARTIFACT_PATTERNS = [
-    'target/kibana-*',
+    'target/kibana-*/**/*.png',
```

Comment:
`target/kibana-*` needs to stay as a pattern. It's grabbing the kibana build tarballs in the `target/` directory.

Was there a problem with `target/kibana-siem/**/*` or `target/kibana-siem/**/*.png`?

Reply:
yeah basically the best thing is maybe 
```
'target/kibana-*',
'target/kibana-siem/**/*.png'
```